open-media-match/src/OpenMediaMatch/app.py

```diff
@@ -152,6 +153,40 @@ def seed_data():
         database.db.session.add(bank)
         database.db.session.commit()
 
+    @app.cli.command("seed_enourmous")
+    def seed_enourmous():
+        """
+        Seed the database with a large number of banks and hashes
+        run command with:
+        export OMM_SEED_BANKS=100
+        export OMM_SEED_HASHES=10000
+        or OMM_SEED_BANKS=100 OMM_SEED_HASHES=10000 flask seed_enourmous
+        It will generate n banks and put n/m hashes on each bank
+        """
+        from threatexchange.signal_type.pdq.signal import PdqSignal
+        from threatexchange.signal_type.md5 import VideoMD5Signal
+        from OpenMediaMatch.storage.interface import BankConfig
```

Comment:
nit: These should be at the top. The lazy loading for the background tasks shouldn't be a general pattern